### PR TITLE
Increase length limit for ID and Name for pipeline (128) and connector (256)

### PR DESCRIPTION
**Pull request by lovromazgon**
### Description

Increases the limits imposed on the length of the connector and pipeline `ID` and `Name` fields to 128 characters.

### Quick checks:

- [X] I have followed the [Code Guidelines](https://github.com/ConduitIO/conduit/blob/main/docs/code_guidelines.md).
- [X] There is no other [pull request](https://github.com/ConduitIO/conduit/pulls) for the same update/change.
- [X] I have written unit tests.
- [X] I have made sure that the PR is of reasonable size and can be easily reviewed.

### CODE REVIEW

File: pkg/connector/service.go [hunk 1]
```diff
@@ -29,6 +29,11 @@ import (
 
 var idRegex = regexp.MustCompile(`^[A-Za-z0-9-_:.]*$`)
 
+const (
+	idLengthLimit   = 128
```
Comment (lyuboxa) [inline pkg/connector/service.go:33]: @lovromazgon can we double the connector limits on the basis that `enrich` will add the pipeline name/id to the connector name. so probably 256+1?

**Review by lyuboxa — APPROVED** [review]

**Review by nassor — APPROVED** [review]

File: pkg/connector/service.go (hunk 1 above)
Comment (simonl2002) [reply pkg/connector/service.go:33]: @lyuboxa is that a platform specific concern?

Comment (lyuboxa) [reply pkg/connector/service.go:33]: @simonl2002 no, part of the pipeline files import

**Review by raulb — APPROVED** [review]

**Review by lovromazgon — COMMENTED** [review]
LGTM @lyuboxa

**Pull request merged**